server/seeder/index.js

```diff
@@ -0,0 +1,51 @@
+import express from 'express';
+import redis from 'redis';
+import client from '../helpers/redis';
+import * as helpers from './helper';
+import data from './placements';
+
+const router = express.Router();
+
+router.get('/place', (req, res) => {
+  client.set('allocations', JSON.stringify(data), redis.print);
+  res.send(data);
+});
+
+const possiblePlacementStatus = [
+  'Placed - Awaiting Onboarding',
+  'External Engagements - Rolling Off',
+];
+
+const placements = () => new Promise((resolve, reject) => {
+  client.get('allocations', (error, result) => {
+    if (error) {
+      console.log(error);
+      reject(error);
+    }
+    return resolve(result);
+  });
+});
+
+const pipeline = async (req, res) => {
+  let values = await placements();
+  if (!values) {
+    values = await placements();
+    if (!values) throw new Error('no data');
+  }
+  const { query: { status } } = req;
+  const jsonValues = JSON.parse(values);
+  if (!status || !possiblePlacementStatus.includes(status)) {
+    return helpers.badResponse(res, { error: 'status query is invalid' });
+  }
+  if (status === 'Placed - Awaiting Onboarding') {
+    return helpers.successResponse(res, { values: jsonValues.onboarding });
+  }
+  if (status === 'External Engagements - Rolling Off') {
+    return helpers.successResponse(res, { values: jsonValues.offboarding });
+  }
```

Comment:
You can reduce the cognitive complexity (if statements) by creating an object as follows:
```
const statuses = {
     'Placed - Awaiting Onboarding': jsonValues.onboarding,
     'External Engagements - Rolling Off': jsonValues.offboarding,
}
return helpers.successResponse(res, { values: statuses[status] }); 
```
Assuming there is a new status called `kickoff` in futute, this would make it easier to extend, without modifying the implementation or without having to add new `if` statements each time

Reply:
Sure, that make sense.